src/tools/m65common.c

```diff
@@ -831,6 +836,8 @@ int push_ram(unsigned long address, unsigned int count, unsigned char* buffer)
       slow_write_safe(fd, cmd, strlen(cmd));
       if (no_rxbuff)
         do_usleep(1000 * SLOW_FACTOR);
+      if (xemu_flag)
```

Comment:
Just my curiosity, why is a longer delay needed with Xemu than with real hardware?

Reply:
I think it came about after discovering an issue prior relating to seeing uart-monitor echo output from xemu being interspersed with serial-output from remotesd prg:
- https://discord.com/channels/719326990221574164/805492126128930846/846537691684143144

The pause I needed on that occasion was quite large:
```
      for (aa = 0; aa < 30000; aa++)
        continue;
```

Perhaps it relates to xemu presently servicing its uart-monitor comms needs only once per frame. (He plans to use a seperate thread for the uart monitor in future versions of xemu, and that may help in this regard).

So for the line above, I think it came about due to a bit of paranoia in me from this earlier discovery, and worried that if you had given the no_rxbuff a bit of extra time for the hardware's needs, then xemu might need even more.

I'm reluctant to remove it for now, suggest we leave it be.

Reply:
That's fine, I was just wondering the reasoning.